api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -48,6 +51,13 @@ public static OpenTelemetry get() {
     if (globalOpenTelemetry == null) {
       synchronized (mutex) {
         if (globalOpenTelemetry == null) {
+
+          OpenTelemetry autoConfigured = maybeAutoConfigure();
+          if (autoConfigured != null) {
+            set(autoConfigured);
+            return autoConfigured;
+          }
+
           OpenTelemetryFactory openTelemetryFactory = Utils.loadSpi(OpenTelemetryFactory.class);
```

Comment:
Unlike the first commit with PoC, I haven't removed the API SPI yet, that we would do in a separate PR.